### PR TITLE
Fix GitHub Version Check

### PR DESCRIPTION
**Pull request by justaduck**
Fix GitHub version check always giving an unable to verify error by checking if message exists since the default function return was None when the function did not exist.

### CODE REVIEW

File: maskgen/ui/masken_ui.py [hunk 1]
```diff
@@ -1357,13 +1357,18 @@ def initCheck(self):
...
+                    if message else ""))
             elif sha_op is not None:
-                tkMessageBox.showinfo('Update to JT Available','New version: {}, Last update message: {}'.format(sha_op, message_op.encode('ascii', errors='xmlcharrefreplace')))
+                tkMessageBox.showinfo('Update to JT Available', 'New version: {}'.format(sha_op) + (
+                    "Last update message: {}".format(message_op.encode('ascii', errors='xmlcharrefreplace'))
+                    if message_op else ""))
         except:
-            tkMessageBox.showwarning('JT Update Status','Unable to verify latest version of JT due to connection error to GitHub. See logs for details')
+            tkMessageBox.showwarning('JT Update Status', 'Unable to verify latest version of JT due to connection '
+                                                         'error to GitHub. See logs for details')
```
Comment (rwgdrummer) [inline maskgen/ui/masken_ui.py]: is this correct.. Looks to be missing a ' + '

File: maskgen/ui/masken_ui.py [hunk 2]
```diff
@@ -1357,13 +1357,18 @@ def initCheck(self):
...
+            sha_op, message_op = OperationsUpdaterGitAPI(branch=git_branch).isOutdated()
             if sha is not None:
-                tkMessageBox.showinfo('Update to JT Available','New version: {}, Last update message: {}'.format(sha, message.encode('ascii', errors='xmlcharrefreplace')))
+                tkMessageBox.showinfo('Update to JT Available', 'New version: {}'.format(sha) + (
+                    " Last update message: {}".format(message.encode('ascii', errors='xmlcharrefreplace'))
+                    if message else ""))
             elif sha_op is not None:
-                tkMessageBox.showinfo('Update to JT Available','New version: {}, Last update message: {}'.format(sha_op, message_op.encode('ascii', errors='xmlcharrefreplace')))
+                tkMessageBox.showinfo('Update to JT Available', 'New version: {}'.format(sha_op) + (
+                    "Last update message: {}".format(message_op.encode('ascii', errors='xmlcharrefreplace'))
```
Comment (rwgdrummer) [inline maskgen/ui/masken_ui.py]: this is a little confusing with mix of "" and ''.

File: maskgen/ui/masken_ui.py (hunk 1 above)
Comment (justaduck) [reply maskgen/ui/masken_ui.py]: The way it sits is correct, even though it looks pretty weird.  It was just converted to cover multiple lines in to follow the 120 line PEP8 standard.

File: maskgen/ui/masken_ui.py [hunk 3]
```diff
@@ -1357,13 +1357,18 @@ def initCheck(self):
...
-            sha_op, message_op= OperationsUpdaterGitAPI(branch=git_branch).isOutdated()
+            sha_op, message_op = OperationsUpdaterGitAPI(branch=git_branch).isOutdated()
             if sha is not None:
-                tkMessageBox.showinfo('Update to JT Available','New version: {}, Last update message: {}'.format(sha, message.encode('ascii', errors='xmlcharrefreplace')))
+                tkMessageBox.showinfo('Update to JT Available', 'New version: {}'.format(sha) + (
+                    ' Last update message: {}'.format(message.encode('ascii', errors='xmlcharrefreplace'))
+                    if message else ''))
             elif sha_op is not None:
-                tkMessageBox.showinfo('Update to JT Available','New version: {}, Last update message: {}'.format(sha_op, message_op.encode('ascii', errors='xmlcharrefreplace')))
+                tkMessageBox.showinfo('Update to JT Available', 'New version: {}'.format(sha_op) + (
```
Comment (rwgdrummer) [inline maskgen/ui/masken_ui.py]: lts get rid of the plus and use one 'format'.
I knoticed this squishes  the version and the 'Last' together

**Pull request merged**